content/webapp/components/CardGrid/index.tsx

```diff
@@ -78,13 +77,18 @@ const CardGrid: FunctionComponent<Props> = ({
                 />
               )}
               {item.type === 'Article' && (
-                <StoryPromoContentApi
+                <StoryPromo
+                  variant="contentApi"
                   article={item}
                   hidePromoText={hidePromoText}
                 />
               )}
               {item.type === 'articles' && (
-                <StoryPromo article={item} hidePromoText={hidePromoText} />
+                <StoryPromo
+                  variant="prismic"
+                  article={item}
+                  hidePromoText={hidePromoText}
+                />
```

Comment:
I'm not sure this is ever used anymore, @davidpmccormick can I pick your brains, did we remove all instances of Grids fetching Prismic stories?